Automattic/wp-calypso
# Help: introduce InlineHelp that uses a Popover to keep users in context

This PR introduces an Inline Help system to Calypso. Previously, clicking the `(?)` at the bottom of the sidebar would directly link to `/help`. We now open a Popover that displays the top four help articles (taken from `/help`) and has basic search functionality. There's also a link to `/help`. 

Current status:

![help popover](https://user-images.githubusercontent.com/191598/35351752-db791514-010f-11e8-8dc3-8096b9431063.gif)

To test:
- Make sure the Popover looks and works well. 
- Make sure the search looks and works well in different situation — when not searching, during search, when finding something, when finding nothing. 
- Make sure the link to `/help` works correctly. 
- Make sure help results with special characters in their title are rendered correctly (e.g. search for "top posts"). 
- Make sure this works the same in different sections of Calypso. 
- Make sure this works and looks fine on mobile. 
- Make sure help can be opened (g, h) and closed (Esc) using the keyboard, and make sure you can navigate (arrow up / down) and select (Enter) search results. 
- Make sure the following interactions send off Tracks events: open / close the Inline Help, search for something, open a search result, open "more help."
